phpstan.neon

```diff
@@ -10,4 +10,5 @@ parameters:
         - '#^Result of static method TorMorten\\Eventy\\Events::#'
         - "#Access to an undefined property#"
         - "#::waitUntilIdle#"
+    checkModelAppends: false
```

Comment:
Is there a reason this needs to be false?
By the looks of it this option solves real bugs 
https://github.com/larastan/larastan/blob/3.x/docs/rules.md#modelappendsrule

Reply:
Yes, I tried to find an alternative but right now larastan will error out on the appends after changing the url function to an attribute cast. Where the functionality is still the same and stuff like indexing still does not work properly when the `appends` is left empty.

Reply:
Ah, you mean
https://github.com/larastan/larastan/issues/1917
https://github.com/larastan/larastan/blob/2.x/docs/features.md#laravel-9-attributes